cellxgene_schema_cli/scripts/gene_processing.py

```diff
@@ -30,59 +34,103 @@ def _parse_gtf(gtf_path: str, output_file: str):
             # Desired features based on whether is gene or transcript
             if line[2] == "gene":
                 features = ["gene_id", "gene_name", "gene_version"]
-            elif line[2] == "transcript":
-                features = ["transcript_id", "transcript_name", "transcript_version"]
+            # TRANSCRIPT implementation
+            #elif line[2] == "transcript":
+            #    features = ["transcript_id", "transcript_name", "transcript_version"]
             else:
                 continue
 
-            # Extract features (column 9 of GTF)
-            current_features = _get_features(line)
+            # Extract features (column 9 of GTF) and get gene length
+            current_features = gtf_tools._get_features(line)
+            current_length = str(gene_lengths[current_features["gene_id"]])
```

Comment:
nit -- can you leave this as an int and then convert it for the string on line 67